src/delagent/agent/util.c

```diff
@@ -591,6 +591,12 @@ void ListFoldersRecurse	(long Parent, int Depth, long Row, int DelFlag)
         LOG_FATAL("folder id=%ld will be deleted with flag %d\n",Parent,DelFlag);
```

Comment:
You could move this line after line 593. This would remove the cluttering of the output of `./delagent --user fossy --password fossy -f`:

```
# Folders
   1 :: Software Repository
      8 :: FolderToDelete
FATAL util.c.591: folder id=8 will be deleted with flag 0

      3 :: test1 (a first test)
         4 :: test1suba (a first subfolder of test1)
FATAL util.c.591: folder id=4 will be deleted with flag 0

FATAL util.c.591: folder id=3 will be deleted with flag 0
[...]
```